database/postgres/postgres.go

```diff
@@ -25,6 +26,7 @@ func init() {
 }
 
 var DefaultMigrationsTable = "schema_migrations"
+var DefaultStatementTimeoutMs = 10000 // 10 seconds
```

Comment:
Make this a `time.Duration` as well

Reply:
sounds good

Reply:
this is actually no longer needed

Reply:
In other words, If the caller
- does not pass any value, we default to 0 (i.e background context)
- passes an invalid value, we return error from Atoi
- passes valid error, we use it as the timeout value